website/src/segment.tsx

```diff
@@ -109,35 +109,40 @@ export const AnnotatedForm = (
     const showSegments = p.viewMode >= ViewMode.Segmentation
     const translation = p.segment.englishGloss.join(", ")
     return (
-      <div className={wordGroup} id={`w${p.segment.index}`}>
-        <div className={syllabaryLayer} lang="chr">
-          {p.segment.source}
-          {p.segment.commentary && p.viewMode >= ViewMode.Pronunciation && (
-            <WordCommentaryInfo commentary={p.segment.commentary} />
+      <a className={anchor} id={`w${p.segment.index}`}>
+        <div className={wordGroup} id={`w${p.segment.index}`}>
```

Comment:
I would consider merging these into one <a> tag or probably instead of making the whole word a link, add a small anchor element next to the word that only shows when it's hovered or focused. I've seen this on many sites where a `#` symbol to the left of a header represents the anchor link.